group-generator.js

```diff
@@ -18,53 +18,49 @@ let period1 = [
   {name: 'Gollum', level: 2}
 ];
 
-
 function addRandomSortNumber(array) {
-  array.map(function(student){
-  student.sortNumber = Math.random();
-
+  array.map(function (student) {
+    student.sortNumber = Math.random();
   });
   return array;
 };
 
 function addLeveledSortNumber(array) {
-  array.map(function(student) {
+  array.map(function (student) {
     student.sortNumber = student.level + Math.random();
-  })
+  });
   return array;
 }
 
 function addOrderedSortNumber(array) {
   let i = 0;
-  array.map(function(student) {
+  array.map(function (student) {
     student.sortNumber = i;
-    i ++;
-  })
+    i++;
```

Comment:
More common to keep a double plus or minus touching the variable it affects, so there's no confusion as to what var is changing.

You'll see this change a few times, too.